packages/client/mutations/AddAgendaItemMutation.ts

```diff
@@ -25,7 +25,10 @@ graphql`
         ...ActionSidebarAgendaItemsSectionAgendaItemPhase @relay(mask: false)
         stages {
           ...ActionMeetingAgendaItemsStage @relay(mask: false)
-          # For useMeetingGotoStageId
+          # For useGotoStageId
+          id
+          isComplete
+          isNavigable
```

Comment:
-1 It is better to make the dependency explicit so the relay compiler can tell us when something is not matching up
```
    meeting {
      ...useGotoStageId_meeting
      phases {
        ...ActionSidebarAgendaItemsSectionAgendaItemPhase @relay(mask: false)
        stages {
          ...ActionMeetingAgendaItemsStage @relay(mask: false)
          isNavigableByFacilitator
        }
      }
    }
  ```

Reply:
@Dschoordsch good point, fixed